blockchain/My-Wallet-V3-iOS
# feat(Lockbox): Resolving IOS-1251 - Now showing lockbox side menu item.

## Objective

The `Lockbox` menu item is now visible.

## Description

I added a `.lockbox` menu item. Also, `SideMenuCell` now has a corresponding xib. If the menu item is not new, the `newContainer` will be hidden. 

## How to Test

1. Build and run
2. Go to the menu
3. See that `Lockbox` is shown along with the `new` label.
4. See that the icons have also been updated. 

## Screenshot/Design assessment

![simulator screen shot - iphone 8 plus - 2018-10-10 at 15 01 00](https://user-images.githubusercontent.com/41585563/46762661-53a42c00-cc9d-11e8-952a-4d1fe9a3cff9.png)

Only other thing I noticed is the peek amount seems to be a bit larger in the comps but, if we want to update that I can do that in a separate PR. 

## Merge Checklist
